cspell.json

```diff
@@ -0,0 +1,93 @@
+{
+  "words": [
+    "ADYEN",
+    "aiohttp",
+    "algoliasearch",
+    "Appsaurus",
+    "asdict",
+    "asyncio",
+    "attributify",
+    "Avalara",
+    "avatax",
+    "backorder",
+    "Behaviour",
+    "Chakra",
+    "clsx",
+    "Codegen",
+    "csrf",
+    "daisyui",
+    "dedup",
+    "Deno",
+    "discogs",
+    "editorjs",
+    "formatjs",
+    "Formset",
+    "hackathon",
+    "hasura",
+    "headlessui",
+    "heroicons",
+    "hmac",
+    "hookform",
+    "interate",
+    "JWKS",
+    "jwst",
+    "Klaviyo",
+    "lerp",
+    "localforage",
+    "mailchimp",
+    "Mailgun",
+    "Mailhog",
+    "mantine",
+    "maxresdefault",
+    "metafield",
+    "middlewares",
+    "Mirumee",
+    "Mjml",
+    "Multiselect",
+    "nuvo",
+    "Offsetted",
+    "overfetching",
+    "pathpida",
+    "pino",
+    "pnpm",
+    "POEDITOR",
+    "pollyjs",
+    "potator",
+    "prefetch",
+    "preorder",
+    "preprocessors",
+    "pytest",
+    "reimagining",
+    "retes",
+    "rudderstack",
+    "saleor",
+    "semibold",
+    "Sendgrid",
+    "starlette",
+    "STARTTLS",
+    "Storyblok",
+    "submenu",
+    "supabase",
+    "sveltejs",
+    "tailwindcss",
+    "tanstack",
+    "taxjar",
+    "testid",
+    "Thirdparty",
+    "tracklist",
+    "Transifex",
+    "trpc",
+    "typeorm",
+    "Undiscounted",
+    "Upstash",
+    "urql",
+    "usehooks",
+    "uvicorn",
+    "VERCEL",
+    "Windi",
+    "windicss",
+    "xstate",
```

Comment:
are these actual wordings? I dont see where we have xstate

Reply:
I checked it to only contain Saleor-specific names and integrations, as it will not scan package.json anymore